File: swagger/models/remove_client_from_class_response.py
from .base_model import BaseModel

from .class import Class

class RemoveClientFromClassResponse(BaseModel):
	"""
	no description available

	Attributes:

		class
		Contains information about the class from which the client was
		removed.
	"""

	input_map = {
		'Class': ('class', Class),
		}

	output_map = {
		'class': 'Class',
		}

		'Class': ('class', Class),

File: swagger/Models/RemoveClientFromClassResponse.php
<?php

namespace App\MindBody\Models;

/**
 * no description available
 */
class RemoveClientFromClassResponse extends BaseModel
{
	/**
	 * Contains information about the class from which the client was
	 * removed.
	 * 
	 * @var Class 
	 */
	public $class;

	protected function getInputMap()
	{
		return [
			'Class' => ['class', Class::class, null],
			];
	}

	protected function getOutputMap()
	{
		return [
			'class' => ['Class', Class::class, null],
			];
	}
}
